packages/overlay/src/integrations/sentry/components/events/EventContexts.tsx

```diff
@@ -31,29 +39,30 @@ export default function EventContexts({ event }: { event: SentryEvent }) {
         </div>
       )}
       <div className="space-y-6">
-        {Object.entries(contexts).map(([ctxKey, ctxValues]) =>
-          ctxValues ? (
-            <div key={ctxKey}>
-              <h2 className="font-bold uppercase">{ctxKey}</h2>
-              <table className="w-full">
-                <tbody>
-                  {Object.entries(ctxValues).map(([key, value]) => (
-                    <tr key={key}>
+        {contextEntries.map(([ctxKey, ctxValues]) => (
+          <div key={ctxKey}>
+            <h2 className="font-bold uppercase">{ctxKey}</h2>
+            <table className="w-full">
+              <tbody>
+                {ctxValues &&
+                  Object.entries(ctxValues).map(([key, value]) => (
+                    <tr key={`${ctxKey}-${key}`}>
                       <th className="text-primary-300 w-1/12 py-0.5 pr-4 text-left font-mono font-normal">
                         <div className="w-full truncate">{key}</div>
                       </th>
                       <td className="py-0.5">
-                        <pre className="text-primary-300 whitespace-nowrap font-mono">
-                          {JSON.stringify(value, undefined, 2)}
-                        </pre>
+                        {typeof value !== 'object' || !value ? (
+                          <pre className="text-primary-300 whitespace-nowrap font-mono">value</pre>
```

Comment:
```suggestion
                          <pre className="text-primary-300 whitespace-nowrap font-mono">{value}</pre>
```

i think you want to render value here when type of value is not object. right?